src/repository/message.py

```
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)